homeassistant/components/apple_tv/remote.py

```diff
@@ -29,6 +33,7 @@
     "volume_up": ("audio", "volume_up"),
     "volume_down": ("audio", "volume_down"),
     "home_hold": ("remote_control", "home"),
+    "select_hold": ("remote_control", "select"),
```

Comment:
Please move the new feature to a separate PR which you open after this fix has been merged.

Reply:
No problem, just wanted to make this process a little faster ;-)

Now the select_hold command is removed for now.